backend/app/readers/create_readers.py

```diff
@@ -0,0 +1,30 @@
+from app.models.settings import Settings
+from app.readers.reader.reader import Reader
+from app.readers.reader.wikipedia_reader import WikipediaReader
+
+
+def create_readers(settings: Settings) -> tuple[Reader | None, ...]:
+    """
+    Return a tuple of Readers depending on the record type in settings.
+    Readers are instantiated with Paths that contain the corresponding record types in the Path names.
+    """
+
+    readers = []
+    if settings.record_types:
+        for record_type in settings.record_types:
+            if (
+                record_type
+                and record_type.lower() == "wikipedia"
```

Comment:
Compare to the enumerator (`RecordType.WIKIPEDIA`) rather than doing string comparisons.

Reply:
Yes, I am using enumerators for comparisons now.